contracts/upgradeable_contracts/multi_amb_erc20_to_erc677/ForeignMultiAMBErc20ToErc677.sol

```diff
@@ -115,6 +115,7 @@ contract ForeignMultiAMBErc20ToErc677 is BasicMultiAMBErc20ToErc677 {
      */
     function bridgeSpecificActionsOnTokenTransfer(ERC677 _token, address _from, uint256 _value, bytes _data) internal {
         if (lock()) return;
+        require(dailyLimit(address(0)) > 0);
```

Comment:
Could you clarify why this check is not located in the `withinLimit` method?

Reply:
I think it is possible to locate this check in the `withinLimit` method. Addressed in the last commit. Thanks for the suggestion.